std/special/compiler_rt/floatunditf.zig

```diff
@@ -0,0 +1,28 @@
+const builtin = @import("builtin");
+const is_test = builtin.is_test;
+const std = @import("../../index.zig");
```

Comment:
Will remove this import and hardcode the mantissa bits where needed to avoid dragging in all of std on test.